rules/sonic-fips.mk

```diff
@@ -0,0 +1,41 @@
+# fips packages
+
+FIPS_VERSION = 0.1
+FIPS_OPENSSH_VERSION = 8.4p1-5+fips
+FIPS_PYTHON_MAIN_VERSION = 3.9
+FIPS_PYTHON_VERSION = 3.9.2-1+fips
+FIPS_GOLANG_MAIN_VERSION = 1.15
+FIPS_GOLANG_VERSION = 1.15.15-1~deb11u2+fips
+BLDENV ?= $(shell lsb_release -cs)
+FIPS_URL_PREFIX = https://sonicstorage.blob.core.windows.net/public/fips/$(BLDENV)/$(FIPS_VERSION)
+
+SYMCRYPT_OPENSSL_NAME = symcrypt-openssl
+SYMCRYPT_OPENSSL = fips/$(SYMCRYPT_OPENSSL_NAME)_$(FIPS_VERSION)_$(CONFIGURED_ARCH).deb
+$(SYMCRYPT_OPENSSL)_SRC_PATH = $(SRC_PATH)/sonic-fips
```

Comment:
>sonic-fips [](http://example.com/codeflow?start=43&length=10)

Do you want to add a new submodule in this PR?

Reply:
No, we will use binaries only, it will take long time to build it.

Reply:
If these packages need to be updated (for fixing security issues, or features/functionality), is there an easy way to rebuild the FIPS packages?

Reply:
Yes, see https://github.com/Azure/sonic-fips.
I will add the artifacts publish automation later, currently, copy the artifacts to storage container manually.

Reply:
which storage? Do you mean FIPS_URL_PREFIX? I notice the name difference: sonic-fips vs fips.

Reply:
Yes, the same some of the other artifacts, sonicstorage.